### PR TITLE
[FLINK-9354][travis] Print execution times for nightly E2E tests

### PR DESCRIPTION
**Pull request by zentol**
## What is the purpose of the change

With this PR we print the execution time for all end-to-end tests.

## Brief change log

* add `start_timer`/`end_timer` functions to `common.sh`
* cover all E2E invocations with timers

## Verifying this change

* observe travis output when execution e2e tests

### CODE REVIEW

**Comment by tzulitai** [comment]
+1 LGTM

**Comment by tzulitai** [comment]
I see these in the Travis logs:
```
flink-end-to-end-tests/run-pre-commit-tests.sh: line 94: start_timer: command not found
...
flink-end-to-end-tests/run-pre-commit-tests.sh: line 97: end_timer: command not found
```

I think you'll have to include `common.sh` in the nightly / pre-commit hook scripts.

File: flink-end-to-end-tests/run-nightly-tests.sh [hunk 1]
```diff
@@ -43,121 +43,151 @@ EXIT_CODE=0
...
 
 if [ $EXIT_CODE == 0 ]; then
   printf "\n==============================================================================\n"
   printf "Running Resuming Savepoint (file, async, scale up) end-to-end test\n"
   printf "==============================================================================\n"
+  start_timer
   STATE_BACKEND_TYPE=file STATE_BACKEND_FILE_ASYNC=true $END_TO_END_DIR/test-scripts/test_resume_savepoint.sh 2 4
   EXIT_CODE=$?
+  end_timer
 fi
```
Comment (tzulitai) [inline flink-end-to-end-tests/run-nightly-tests.sh]: Perhaps we can refactor this whole block into a common base script for the nightly / pre-commit hook scripts.
The end_timer / start_timer functions should also be located there.

**Comment by zentol** [comment]
nice catch @tzulitai, will rework this into a run_test method that takes a description and command to execute.

File: flink-end-to-end-tests/test-scripts/test_resume_savepoint.sh [hunk 2]
```diff
@@ -33,8 +33,8 @@ else
   NUM_SLOTS=$NEW_DOP
 fi
 
-STATE_BACKEND_TYPE=${STATE_BACKEND_TYPE:-file}
-STATE_BACKEND_FILE_ASYNC=${STATE_BACKEND_FILE_ASYNC:-true}
+STATE_BACKEND_TYPE=${3:-file}
+STATE_BACKEND_FILE_ASYNC=${4:-true}
```
Comment (tzulitai) [inline flink-end-to-end-tests/test-scripts/test_resume_savepoint.sh]: We should also update the usage message at the beginning of this file to reflect these extra parameters.

Comment (zentol) [reply flink-end-to-end-tests/test-scripts/test_resume_savepoint.sh]: will do 👍

**Pull request closed without merging**